src/components/filter.js

```diff
@@ -78,21 +79,24 @@ const Filter = ({
             )
         }
       })}
-      <ListItem
-        innerDivStyle={style.listItemInner}
-        leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
+      <FlatButton
+        label={intl.formatMessage(messages.changeFilters)}
```

Comment:
Let's use <FormattedMessage> here to keep consistent with other code and avoid the extra code of injecting intl